Controllers/Users.controller.js

```diff
@@ -0,0 +1,10 @@
+const selectUsers = require("../Models/Users.model.js");
+
+const getUsers = (req, res) => {
+  selectUsers().then((users) => {
+        res.status(200).send(users);
```

Comment:
Needs to be on a key of users

Reply:
Im not sure if i know what you mean by this , could you clarify please?